### PR TITLE
feat: Decentraland Wearable Rarity [decentraland-wearable-rarity]

### PR DESCRIPTION
**Pull request by 2fd**
Changes proposed in this pull request:
- creates `decentraland-wearable-rarity` strategy
- prevents `decentraland-estate-size` to fail if and invalid chain is selected
- adds `README` to `decentraland-estate-size`

### CODE REVIEW

File: src/strategies/decentraland-wearable-rarity/index.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+export const author = '2fd';
+export const version = '0.1.0';
+
+const DECENTRALAND_COLLECTIONS_SUBGRAPH_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/decentraland/collections-ethereum-mainnet',
+  '3': 'https://api.thegraph.com/subgraphs/name/decentraland/collections-ethereum-ropsten',
+  '137': 'https://api.thegraph.com/subgraphs/name/decentraland/collections-matic-mainnet',
```
Comment (nachomazzara) [inline src/strategies/decentraland-wearable-rarity/index.ts:10]: We only need the Ethereum one

File: src/strategies/decentraland-wearable-rarity/index.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    return scores
+  }
+
+  // initialize multiplers and params
+  const multiplers = options.multipliers || {  };
+  const params = {
+    nfts: {
+      __args: {
+        where: {
+          itemType_in: ['wearable_v1',  'wearable_v2', 'smart_wearable_v1'],
```
Comment (nachomazzara) [inline src/strategies/decentraland-wearable-rarity/index.ts:40]: We need just `wearables_v1`

Comment (nachomazzara) [inline src/strategies/decentraland-wearable-rarity/index.ts:40]: We must filter by contract address too. We are not going to compute all the L1 wearables, just a few of them

File: src/strategies/decentraland-estate-size/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Decentraland Wearable Rarity
+
+This strategy allows users to calculate the total amount of Decentraland lands in estates and apply a multiplier to the result.
+
+## Example
+
+The should should look like this:
```
Comment (andyesp) [inline src/strategies/decentraland-estate-size/README.md:7]: Maybe "The _parameters_ should look like this"?

File: src/strategies/decentraland-wearable-rarity/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        "legendary": 10000,
+        "epic": 1000,
+        "rare": 100,
+        "uncommon": 10,
+        "common": 1
+    }
+}
+```
+
+If you want to restrict which collection is take into account should looks like this:
```
Comment (andyesp) [inline src/strategies/decentraland-wearable-rarity/README.md:24]: Typos and change in grammar? "If you want to restrict which collection is taken into account, parameters should looks like this:"

File: src/strategies/decentraland-wearable-rarity/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+# Decentraland Wearable Rarity
+
+This strategy allows users to calculate the total amount of Decentraland wearables and apply a multiply to each rarity. Additionally you can limit the collections that will be take into account using a list of ids.
+
+## Example
+
+The should should look like this:
```
Comment (andyesp) [inline src/strategies/decentraland-wearable-rarity/README.md:7]: Here also "The parameters should look like this"

File: src/strategies/decentraland-wearable-rarity/README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+# Decentraland Wearable Rarity
+
+This strategy allows users to calculate the total amount of Decentraland wearables and apply a multiply to each rarity. Additionally you can limit the collections that will be take into account using a list of ids.
```
Comment (andyesp) [inline src/strategies/decentraland-wearable-rarity/README.md:3]: Typo, change "multiply" to "multiplier".
Also "that will be taken into account".

**Review by 1emu — COMMENTED** [review]
LGTM

**Review by nachomazzara — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**